lib/engine/game/base.rb

```diff
@@ -330,8 +330,10 @@ def self.load_from_json(*jsons)
         const_set(:LAYOUT, data['layout'].to_sym)
       end
 
-      def initialize(names, id: 0, actions: [], pin: nil, strict: false, optional_rules: [])
+      def initialize(names, id: 0, owner: nil, actions: [], pin: nil, strict: false, optional_rules: [])
```

Comment:
i don't like this

Reply:
Where would you put it? The game needs to know who is its owner somehow, so it can tell the other components

Reply:
i think it's a bit strange, if we're going to do something like this i'd prefer something less invasive. 

can we remove the concept of owner as a fake player, and instead have that be inferred? 

somehow allowing you to make actions but not having to have owner/owner_player within the engine

Reply:
The owner is not something new I'm adding, it was already there but was only being used for new games. I'm just making the game aware of it and allowing that user to act.

This is only being used in actionable and game_log (which includes actionable). In actionable we have @game and @game_data. A workaround would be to not pass the owner to @game and just use the information from @game_data.

I think any other solution would be more invasive, because we'd always need to have the game aware of it somehow.